soen341-UB3-QA/resources/views/index.blade.php

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
```

Comment:
This view is not used anywhere. It is not needed either. You can remove this file.